plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java

```diff
@@ -15,6 +15,7 @@
 
```

Comment:
commit message:
`Support reading of Shallow Cloned tables` -> `Add handling for absolute paths in the Delta transaction log`